### DISCUSSION TITLE
Questions regarding Vertical & Horizontal Shells

### DISCUSSION THREAD

**Discussion by BuckyTheElk** [supermerill/SuperSlicer#2199] [Q&A]
So I am somewhat new to SuperSlicer. It's great. I just had a question about the horizontal shell minimum thickness. I typically use 3 perimeters when printing (for top and bottom too) and noticed that under the vertical shells section SS recommends a 2.23mm minimum wall thickness for 3 perimeters. 

Should I input this value into the minimum shell thickness top and bottom sections?

**Comment by supermerill** [comment]
I don't advise to go below 3 top layers.
Do as you want, top/bottom isn't the same as perimeters, it's not isotropic.
